ada_feeding/ada_feeding/trees/move_to_tree.py

```diff
@@ -67,7 +62,12 @@ def create_tree(
         tree: The behavior tree that moves the robot above the plate.
         """
         self.create_blackboard(name)
-        return self.create_move_to_tree(name, logger, node)
+
+        # Import the action type
+        self.action_type_class = import_from_string(action_type_class_str)
```

Comment:
Not sure how I didn't catch this, but I think this breaks the code (it should be `action_type`, not `action_type_class_str`). I'm currently working on a large linting PR and will fix this as part of that.

Reply:
Addressed in #47 , I will test it and then tag you for review.